### PR TITLE
Book list

### PR DESCRIPTION
**Pull request by seromarin**
## Summary
- Completed the book list

## Screenshot
Captura de Pantalla 2020-10-30 a la(s) 4 57 01 p  m

## Trello Card
https://trello.com/c/8akclPNT/30-listado-de-libros

### CODE REVIEW

File: maquetacion/index.html [hunk 1]
```diff
@@ -10,30 +10,99 @@
...
-        </div>
-        <div class="detail__info">
-          <div class="detail__author">
-            <h3 class="text-bold text-20">Autor del libro:</h3>
-            <p class="text-light text-20">Nombre del autor del libro</p>
-          </div>
-          <div class="detail__editorial">
-            <h3 class="text-bold text-20">Editorial:</h3>
-            <p class="text-light text-20">Nombre de la editorial</p>
+
```
Comment (idcuadrowolox) [inline maquetacion/index.html]: Please remove empty line

File: maquetacion/index.html [hunk 2]
```diff
@@ -10,30 +10,99 @@
...
-          <div class="detail__author">
-            <h3 class="text-bold text-20">Autor del libro:</h3>
-            <p class="text-light text-20">Nombre del autor del libro</p>
-          </div>
-          <div class="detail__editorial">
-            <h3 class="text-bold text-20">Editorial:</h3>
-            <p class="text-light text-20">Nombre de la editorial</p>
+    
+    <div class="container">
+
```
Comment (idcuadrowolox) [inline maquetacion/index.html]: same

File: maquetacion/index.html [hunk 3]
```diff
@@ -10,30 +10,99 @@
...
+      <div id="detail" class="detail">
+
```
Comment (idcuadrowolox) [inline maquetacion/index.html]: same

File: maquetacion/index.html [hunk 4]
```diff
@@ -10,30 +10,99 @@
...
+        <div class="detail__cover">
+          <img class="detail__cover-img" src="./assets/book-cover.png" alt="Book Wolox Cover">
+          <img class="detail__cover-badge" src="./assets/badge.png" alt="Book Badge">
+        </div>
+
```
Comment (idcuadrowolox) [inline maquetacion/index.html]: Please remove all empty lines

File: maquetacion/src/scss/_styles.scss [hunk 5]
```diff
@@ -6,18 +6,16 @@
...
-.text-20 {
-  font-size: 1.25rem;
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin: 0 auto;
+  width: 1024px;
+  // height: 100%; Enable later
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_styles.scss]: remove comment

File: maquetacion/src/scss/_book-list.scss [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    margin: 50px 0;
+  }
+
+  &__author {
+    color: #828282;
+    font-family: Roboto;
+    font-size: 14px;
+    height: 14px;
+    line-height: 18px;
+    width: 109px;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_book-list.scss]:
```suggestion
    width: 100%;
```

File: maquetacion/src/scss/_book-list.scss [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    width: 142px;
+  }
+
+  &__title {
+    font-size: 16px;
+    font-weight: 700;
+    height: 15px;
+    margin: 6px 0;
+    line-height: 18px;
+    width: 109px;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_book-list.scss]: It's not necessary

File: maquetacion/src/scss/_detail.scss [hunk 8]
```diff
@@ -1,29 +1,92 @@
+@import 'colors';
+
+%text-title {
+  font-size: 20px;
+  font-weight: 700;
+}
+
+%text-content {
+  font-size: 20px;
+  color: $grey-light;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss:9]: Sort properties by name

File: maquetacion/src/scss/_detail.scss [hunk 9]
```diff
@@ -1,29 +1,92 @@
...
+  }
+
+  &__book-genre {
+    color: $grey-light;
+    font-family: Roboto;
+    font-size: 24px;
+    font-weight: 700;
+    height: 30px;
+    line-height: 28px;
+    width: 98px;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss:35]: Remove this

**Review by idcuadrowolox — CHANGES_REQUESTED** [review]

File: maquetacion/src/scss/_detail.scss (hunk 9 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:35]: Removed

File: maquetacion/src/scss/_detail.scss (hunk 8 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:9]: Sorted in next commit

**Review by seromarin — COMMENTED** [review]
All fixes are applied

**Review by idcuadrowolox — APPROVED** [review]

**Pull request merged**